platform-ui-skin/src/main/webapp/skin/less/social/skin/HamburgerMenu/Style.less

```diff
@@ -20,7 +20,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 .iconSnapshot:before {
   content: "\eb67";
 }
-.iconOverview:before {
+.iconOverview:before, .iconParticipate:before {
```

Comment:
There are two solutions:

- Use "overview" as node name with "Participate" as english label, in that case no need to add a new css for the icon
- Use "participate" as node name, in that case IMO the css should go instead to dw less file since we will add a node in eXo package and not a Meeds one.

In the both cases, we should move .iconSnapshot:before in dw less file for the same last reason.
cc @hakermi @mkrout WDYT ?

Reply:
@azayati Thanks for the proposition, i changed the node name to overview.